e2e-tests/insurance/cypress/e2e/journeys/check-your-answers/export-contract/change-your-answers/how-the-contract-was-awarded/change-your-answers-how-the-contract-was-awarded-from-other-method.spec.js

```diff
@@ -60,7 +60,7 @@ context(
         beforeEach(() => {
           cy.navigateToUrl(url);
 
-          // select DIRECT_AWARD
+          // change to DIRECT_AWARD
```

Comment:
was it not meant to be select?

Reply:
Only the second ones, the first ones call `completeAndSubmitHowWasTheContractAwardedForm` whereas the second instances just select the radio, rather than submitting the answer